### PR TITLE
add types, testing and CI

### LINKED ISSUE #14: type checking

### PR DESCRIPTION
**Pull request by bartveneman**
closes #14 
closes #19

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// @ts-expect-error Typing of css-tree is incomplete
```
Comment (b-vb) [inline index.js:1]: Props for expect error with description instead of ignore

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (b-vb) [inline tsconfig.json:1]: Looks good, TS docs are pretty great, but I've also found [this video](https://www.youtube.com/watch?v=eJXVEju3XLM&t=1s) helping

File: index.js (hunk 1 above)
Comment (bartveneman) [reply index.js:1]: Learned it from the best! 😉

File: tsconfig.json (hunk 2 above)
Comment (bartveneman) [reply tsconfig.json:1]: Amazing, very good resource, thanks!

**Pull request merged**